diff --git a/vendor/tomatophp/filament-payments/config/filament-payments.php b/vendor/tomatophp/filament-payments/config/filament-payments.php
index 134da0f..556eb3f 100644
--- a/vendor/tomatophp/filament-payments/config/filament-payments.php
+++ b/vendor/tomatophp/filament-payments/config/filament-payments.php
@@ -6,7 +6,7 @@ return [
         \TomatoPHP\FilamentPayments\Services\Drivers\Moyaser::class,
         \TomatoPHP\FilamentPayments\Services\Drivers\Payfort::class,
         \TomatoPHP\FilamentPayments\Services\Drivers\Paymob::class,
-        \TomatoPHP\FilamentPayments\Services\Drivers\PayPal::class,
+        \TomatoPHP\FilamentPayments\Services\Drivers\Paypal::class,
         \TomatoPHP\FilamentPayments\Services\Drivers\Paytabs::class,
         \TomatoPHP\FilamentPayments\Services\Drivers\Plisio::class,
         \TomatoPHP\FilamentPayments\Services\Drivers\StripeV3::class,
diff --git a/vendor/tomatophp/filament-payments/src/Filament/Pages/PaymentGateway.php b/vendor/tomatophp/filament-payments/src/Filament/Pages/PaymentGateway.php
index 69b259e..a9fb659 100644
--- a/vendor/tomatophp/filament-payments/src/Filament/Pages/PaymentGateway.php
+++ b/vendor/tomatophp/filament-payments/src/Filament/Pages/PaymentGateway.php
@@ -35,7 +35,7 @@ class PaymentGateway extends Page implements Tables\Contracts\HasTable
 
     protected ?string $status = null;
 
-    protected static ?string $navigationIcon = 'heroicon-o-cog';
+    protected static string | \BackedEnum | null $navigationIcon = 'heroicon-o-cog';
 
     protected static string $view = "filament-payments::pages.payment-gateway";
 
diff --git a/vendor/tomatophp/filament-payments/src/Filament/Resources/PaymentResource.php b/vendor/tomatophp/filament-payments/src/Filament/Resources/PaymentResource.php
index 1a29883..bb72174 100644
--- a/vendor/tomatophp/filament-payments/src/Filament/Resources/PaymentResource.php
+++ b/vendor/tomatophp/filament-payments/src/Filament/Resources/PaymentResource.php
@@ -24,7 +24,7 @@ class PaymentResource extends Resource
 {
     protected static ?string $model = Payment::class;
 
-    protected static ?string $navigationIcon = 'heroicon-o-banknotes';
+    protected static string | \BackedEnum | null $navigationIcon = 'heroicon-o-banknotes';
 
     protected static ?int $navigationSort = 1;
 
@@ -113,7 +113,7 @@ class PaymentResource extends Resource
             ->actions([
-                Tables\Actions\ViewAction::make(),
+                ViewAction::make(),
             ])
             ->bulkActions([
                 // Tables\Actions\ViewAction::make(),
